### PR TITLE
#1056 added examples on how to pass custom drive cycle

### LINKED ISSUE #1056: Extend parameters-cli for drive cycles
**Summary**
Extend the capabilities of parameters-cli (or create a new function) so the drive cycle folder can be copied to a local directory and new drive cycles added locally.

### PR DESCRIPTION
**Pull request by ferranbrosa**
# Description

Added examples in the corresponding notebooks on how to pass a custom drive cycle to PyBaMM. Given that it is a change in the examples only, I didn't add any line to CHANGELOG.

Fixes #1056

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/master/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [x] New feature (non-breaking change which adds functionality)

# Key checklist:

- [x] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [x] Code is commented, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by ferranbrosa** [comment]
> Looks good to me, thanks! I think that now that this works, it should be the only way to specify drive cycles (i.e. our examples should only show that way). And can you adapt `Simulation` to figure out that if current is an `Interpolant`, it should use the "drive cycle" operating mode and so passing `t_eval` is not required?

When you say that it should be the only way to specify drive cycles you mean to remove the examples where we do `"[current data]US06"`?

**Comment by tinosulzer** [comment]
Yes, that's what I mean (@rtimms  do you agree?). And make it clear that the drive cycle data can be anywhere, `inputs/drive_cycles` is just where we happen to store the examples

**Comment by ferranbrosa** [comment]
Shall I just remove the functionality or place a deprecation warning?

**Comment by rtimms** [comment]
Yeah, I agree. It makes it much easier for people to use their own drive cycle data and gives you more control over the interpolant, which is nice. Thanks @ferranbrosa !

**Comment by ferranbrosa** [comment]
I have now implemented Tino's comments. I haven't removed the old functionality or added a deprecation warning but it should be fairly easy to do.

File: examples/notebooks/Getting Started/Tutorial 4 - Setting parameter values.ipynb [hunk 1]
```diff
@@ -18,17 +18,9 @@
...
-     "output_type": "stream",
-     "text": [
-      "Note: you may need to restart the kernel to use updated packages.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "%pip install pybamm -q    # install PyBaMM if it is not installed\n",
+    "#%pip install pybamm -q    # install PyBaMM if it is not installed\n",
```
Comment (tinosulzer) [inline examples/notebooks/Getting Started/Tutorial 4 - Setting parameter values.ipynb:23]: why has this been commented out?

Comment (ferranbrosa) [reply examples/notebooks/Getting Started/Tutorial 4 - Setting parameter values.ipynb]: That was a typo, thanks!

**Pull request merged**